DEVELOPMENT.md

```diff
@@ -183,25 +227,22 @@ npm run typecheck
 
 You can test these packages locally by linking them, or by using a local npm registry such as Verdaccio.
 
-#### Using linking (recommended for local development)
+#### Package linking
```

Comment:
1. Can _any_ package be linked like this for local use or only CLI packages?

2. At least `follow steps 1-4`, no?

3. Why does this `npm link` have a `-w` ? Can it be deduped into `follow steps 1-5` or should it remain separate (E.g. if it's always needed we should added it to the top instructions)?